### DISCUSSION TITLE
Error 403

### DISCUSSION THREAD

**Discussion by sfalvey** [amberelectric/public-api#171] [Q&A] [answered]
Bit confused - wondering if there is something wrong with my account but after enabling an API key and authenticating on the developers webpage, I cant seem to list my sites. The return string is.

  "message": "User is not authorized to access this resource with an explicit deny"

I can confirm that the authorize panel does say I am authorized. I have tried logging out and back in but no luck. I have also tried using curl and the "try it now" options.

**Comment by madpilot** [comment]
That error usually means that the URL is wrong. Can you cut and paste your cURL command? (redact your API key)

**Comment by sfalvey** [comment]
As generated on the developer page.....

curl -X 'GET' \
  'https://api.amber.com.au/v1/sites' \
  -H 'accept: application/json' \
  -H 'Authorization: Bearer apikey'

and when executed directly from the web page: 

![Screenshot 2024-01-30 at 11 28 47 am](https://github.com/amberelectric/public-api/assets/284672/e16e2d57-2e15-4d83-9157-bf6ba079c83a)

**Reply by madpilot** [reply, marked as answer]
Double check you cut and pasted your API key correctly - it should start with `psk_` - sometimes cut and paste misses the first bit.

**Reply by sfalvey** [reply]
Thanks - yes that was it. The c&p missed the psk_. All working now :)
